stories/clickable_link.fmf

```diff
@@ -0,0 +1,45 @@
+/clickable_test_link
+    story:
+        As a tester I want to link test verifying Jira issue.
+
+    description:
+        User can click on a link in JIRA issue that returns human-readable metadata of test that covers
+        this issue.
+
+    example:
+      - |
+        url-options: ?test_url=https://github.com/teemtee/tmt&test-name=/tests/core/smoke
+
+    link:
+      - https://issues.redhat.com/browse/TT-207
+
+/clickable_plan_link
+    story:
+        As a tester I want to link plan verifying Jira issue.
+
+    description:
+        User can click on a link in JIRA issue that returns human-readable metadata of plan that covers
+        this issue.
```

Comment:
Here the description should state that all tests executed under given plan are covering the issue.